### ISSUE TITLE
jsPsych studies should receive child/session info

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1294]
# TL;DR

Researchers running jsPsych experiments should have access to the same child and session information that they currently do when using EFP using a protocol generator function.

# Narrative

As a researcher, I'd like access to certain bits of information about the child and their previous session data in the study code. This allows me to create an experiment that can change based on the child's information and their participation/performance history (e.g. for multi-session/longitudinal designs and adaptive difficulty methods).

# Acceptance Criteria

- [ ] Tests
- [ ] Scoping
- [ ] Documentation
- [ ] Should include, at a minimum, the same information and format that we use in the EFP's [protocol generator function](https://lookit.readthedocs.io/projects/frameplayer/en/latest/utils/protocol-generator.html#what-information-it-has-access-to)
